src/vanilla/Model/CodeModelPy.cs

```diff
@@ -132,10 +133,7 @@ public virtual string ConfigConstructorParameters
                     configParams.Add(property.Name);
                 }
                 var param = string.Join(", ", configParams);
-                if (!IsCustomBaseUri)
-                {
-                    param += (param.IsNullOrEmpty() ? "" : ", ") + "base_url";
-                }
+                param += (param.IsNullOrEmpty() ? "" : ", ");
```

Comment:
Why?

Reply:
We changed the way to handle CustomBaseUri. Now we handle it in Service Client. If it is a custom base url, we add base_url = '@Model.BaseUrl'. You can find the code in ServiceClientTemplate.cshtml